assets/src/conversion/index.js

```diff
@@ -104,130 +114,115 @@ class Conversion extends Component {
-					</p>
-					<ul>
-						<li>
-							{ __( 'conversion batches processed so far:' ) }{' '}
-							<b>
-								<u>{ queuedBatchesCsv }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'total batches queued:' ) }{' '}
-							<b>
-								<u>{ maxBatch }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'conversion batch size (posts per batch):' ) }{' '}
-							<b>
-								<u>{ conversionBatchSize }</u>
-							</b>
-						</li>
-					</ul>
-
-					<br />
-
-					<h3>{ __( 'Reset Ongoing Conversion to Start Again' ) }</h3>
-					<p>
-						{ __(
-							'In case that your active conversion browser tab has been closed by accident, or it has been interrupted and closed unexpectedly, you may reset the conversion status here, and start converting all over again. Note that this will enable you to restart the conversion, but any previous results may be lost.'
-						) }
-					</p>
-					<input
-						type="submit"
-						className="large"
-						value={ __( 'Reset Conversion' ) }
-						onClick={ this.handleOnClickResetConversion }
-					/>
-				</div>
-			);
-		} else {
-			return (
-				<div className="ncc-page">
-					<h1>{ __( 'Run Conversion' ) }</h1>
-
-					<p>
-						<b>
-							<u>{ __( 'Note carefully' ) }</u>
-						</b>
-						{ ' -- ' }
-						{ __(
-							'once started, the conversion should not be interrupted! Your browser page needs to remain active until conversion is complete. You may, however, manually open multiple browser tabs once the conversion starts, and each of them will fetch and convert another batch of posts in parallel. Running multiple tabs speeds up the conversion. Recommended max number of tabs is 4.'
-						) }
-					</p>
-
-					<h3>{ __( 'Planned Conversion' ) }</h3>
-					<ul>
-						<li>{ __( 'the entire queued content will be converted to Gutenberg Blocks' ) }</li>
-						<li>{ __( 'this page will automatically reload for every batch' ) }</li>
-						<li>
-							{ __( 'number of posts/entries to be converted:' ) }{' '}
-							<b>
-								<u>{ queuedEntries }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'total conversion batches queued' ) }:{' '}
-							<b>
-								<u>{ maxBatch }</u>
-							</b>
-						</li>
-						<li>
-							{ __( 'batch size (posts per batch):' ) }{' '}
-							<b>
-								<u>{ conversionBatchSize }</u>
-							</b>
-						</li>
-					</ul>
-
-					<br />
-
-					<input
-						type="submit"
-						className="large"
-						value={ __( 'Start Conversion' ) }
-						onClick={ this.handleOnClickInitializeConversion }
-					/>
-
-					<br />
-					<br />
-
-					{ !! someConversionsFailed && (
-						<div>
-							<h3>{ __( 'Retry Converting Failed Posts' ) }</h3>
+				<Fragment>
+					<div className="newspack-logo-wrapper">
+						<NewspackLogo />
+					</div>
+					<Grid>
+						<FormattedHeader
+							headerIcon={ <UnarchiveIcon /> }
+							headerText={ __( 'Run conversion' ) }
+							subHeaderText={ __( 'A converson is currently in progress.' ) }
```

Comment:
Proposing change in text:
Start conversion to Gutenberg Blocks
(this keeps the sub-title the same as in the other case -- the page looks more recognizable and simpler like that, when the subtitle is not changing unnecessarily)

Reply:
Fixed in ec06523